src/main/kotlin/com/intershop/gradle/isml/IsmlPlugin.kt

```diff
@@ -169,12 +162,8 @@ open class IsmlPlugin : Plugin<Project> {
                 }
     }
 
```

Comment:
I know the your code is shorter, but please use the longer code. With this code it is easier to identify problems.